### DISCUSSION TITLE
More sophisticated axis shaping

### DISCUSSION THREAD

**Discussion by sbman** [bwRavencl/ControllerBuddy#175] [Ideas]
I found Controllerbuddy and was hoping I could use it to scale the controller sticks for games that have really poor axis mapping and  deadband issues.  The idea is to take the raw value of the stick input and apply a curve to the vjoy axix output so that you can get non linear outputs into the game to either make the controller more sensitive, or with the opposite curve applied, less sensitive.  Being able to set limits to the endpoints would be super helpful too for some situations. I know flight sims usually have reasonable editors for these types of setups, but many other games do not.

**Comment by bwRavencl** [comment]
Hello @sbman,

axis shaping is already implemented via the *Exponent* setting of axis actions.
There is even a little visualization displaying the curve.
What is, currently not supported are exponents smaller than one (or rather between 0 and 1) to make an axis more sensitive around the center.
I will take a look at this shortly...

Setting a lower and upper bound for axis action outputs is also currently not supported, simply because it was something I personally never needed, but I am willing to take a look at this as well.

**Comment by bwRavencl** [comment]
The requested additional axis shaping features are now implemented and released with ControllerBuddy-1.2.0.
